docs/AWS.md

```diff
@@ -0,0 +1,147 @@
+aws ec2 create-security-group \
+  --group-name CalicoSG \
+  --description CalicoSecurityGroup
+aws ec2 authorize-security-group-ingress \
+  --group-name CalicoSG \
+  --protocol tcp \
+  --port 22 \
+  --source-group 0.0.0.0/0
+aws ec2 authorize-security-group-ingress \
+  --group-name CalicoSG \
+  --protocol all \
+  --port all \
+  --source-group CalicoSG
+```
+
+## Spinning up the VMs
+etcd needs to be running on the Calico hosts.  The easiest way to bootstrap etcd is with a discovery URL.  Choose an etcd cluster size that is equal to or less than the number of Calico nodes (an odd number in the range 3-9 works well).  We'll use 3 for the size of the etcd cluster and the Calico nodes in the instructions below.  Use `curl` to get a fresh discovery URL (replace size=3 with your cluster size if desired):
+```
+curl https://discovery.etcd.io/new?size=3
+```
+You need to grab a fresh URL each time you bootstrap a cluster.
+
+Create a file `cloud-config.yaml` with the following contents; **replace `<discovery URL>` with the URL retrieved above**:
+```
+#cloud-config
+coreos:
+  update:
+    reboot-strategy: off
+  etcd2:
+    name: $private_ipv4
+    discovery: <discovery URL>
+    advertise-client-urls: http://$private_ipv4:2379
+    initial-advertise-peer-urls: http://$private_ipv4:2380
+    listen-client-urls: http://0.0.0.0:2379,http://0.0.0.0:4001
+    listen-peer-urls: http://$private_ipv4:2380,http://$private_ipv4:7001
+  units:
+    - name: etcd2.service
+      command: start
+
+```
+Note: we disable automated reboots for this demo to avoid interrupting the instructions.
+
+Then create the cluster with the following command:
+```
+aws ec2 run-instances \
+  --count 3 \
+  --image-id ami-29734819 \
+  --instance-type t1.micro \
+  --key-name calicokey \
+  --security-groups CalicoSG \
+  --user-data file://cloud-config.yaml
+```
+
+## Installing calicoctl on each node
+Get the public IP addresses of the new instances from the AWS Web Console or by running:
+```
+aws ec2 describe-instances --filter "Name=key-name,Values=calicokey" | grep PublicIpAddress
+```
+
+SSH into each node and run these commands to set up Calico:
+```
+# SSH into a node with the calicokey.pem and username core
+ssh -i calicokey.pem core@<instance IP>
+
+# Download calicoctl and make it executable:
+wget https://github.com/Metaswitch/calico-docker/releases/download/v0.4.5/calicoctl
+chmod +x ./calicoctl
+
+# Grab our private IP from the metadata service:
+export metadata_url="http://169.254.169.254/latest/meta-data"
+export private_ip=$(curl "$metadata_url/local-ipv4")
+
+# Start the calico node service:
+sudo ./calicoctl node --ip=$private_ip
+```
+Then, on any one of the hosts, run this command to create an IP pool with IP-in-IP and NAT enabled:
+```
+./calicoctl pool add 192.168.0.0/16 --ipip --nat-outgoing
+```
+IP-in-IP alows Calico to route traffic between containers.  NAT allows the containers to make outgoing connections to the internet.
```

Comment:
Per previous comment, I think you should delete this.  There are lots of parameters that are equally important that we aren't explaining in detail.